### PR TITLE
Fix/327 plant search in frontend

### PR DESCRIPTION
**Pull request by badnames**
## Basics

- [ ] Details of what you changed are in commit messages.
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Comment by markus2330** [comment]
Great to see you are working on this important topic!

Please always make sure to branch off from the latest master. This PR already has conflicting files.

File: :w
Comment (markus2330) [inline :w:1]: I thought that only happens to me (typing `:w` twice in vim) :laughing:

File: frontend/package.json [hunk 1]
```diff
@@ -80,6 +80,7 @@
     "autoprefixer": "^10.4.13",
     "babel-jest": "^29.5.0",
     "babel-loader": "^8.3.0",
+    "esbuild": "^0.17.19",
```
Comment (markus2330) [inline frontend/package.json:83]: Don't we have a bundler yet? Please avoid other changes so that your PRs can get merged faster.

File: package-lock.json
Comment (markus2330) [inline package-lock.json:1]: Something went wrong here, too.

**Review by markus2330 — CHANGES_REQUESTED** [review]
Only had a very quick look. Feel free to ignore the comments if it was too early.

File: frontend/package.json (hunk 1 above)
Comment (badnames) [reply frontend/package.json:83]: That was added accidentally while resolving a local build issue.

**Comment by badnames** [comment]
> Great to see you are working on this important topic!
> 
> Please always make sure to branch off from the latest master. This PR already has conflicting files.

I created this branch in GitHub. It's probably outdated because I started to work on this PR one day after creating the branch.

**Comment by badnames** [comment]
@markus2330 This PR still contains a styling problem that might be a candidate for a FLOSS issue:

The plant search component does not match the width of its parent and the margins of the search icon are too small.
image

I can also fix it now if this is more important right now, just let me know.

**Review by markus2330 — CHANGES_REQUESTED** [review]
Thanks for the PR! Please have a look at your own changes in the PR, there are several wrong/irrelevant changes here. Furthermore, there is a merge conflict. It is probably best if you start a new branch and only apply the relevant changes there.

PRs ideally shouldn't have open issues, except if it is something hard to implement or the PR already gets to big. In such cases, please create a follow-up issue.

**Pull request closed without merging**